docs/tasks/access-kubernetes-api/migrate-third-party-resource.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Migrate a ThirdPartyResource to CustomResourceDefinition
```

Comment:
nit: should this be "Migrate a ThirdPartyResource to **a** CustomResourceDefinition"?